package.json

```diff
@@ -225,7 +229,8 @@
         "visible": true,
         "subPermissions": [
           "orders.item.delete",
-          "orders.po-lines.item.delete"
+          "orders.po-lines.item.delete",
+          "orders.routing-lists.item.delete"
```

Comment:
I haven't managed to find the action on UI to delete a routing list, is there one?

Reply:
As we agreed with POs' the delete functionality will be added fo the scope of the ticket